hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -393,9 +393,13 @@ public MasterRpcServices(HMaster m) throws IOException {
   }
 
   @Override
-  protected RpcServerInterface createRpcServer(Server server, Configuration conf,
-      RpcSchedulerFactory rpcSchedulerFactory, InetSocketAddress bindAddress, String name)
-      throws IOException {
+  protected RpcServerInterface createRpcServer(
```

Comment:
@saintstack @binlijin Here's an example of this style with a throws clause.

Reply:
OK